src/App.js

```diff
@@ -1,3 +1,67 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import InputDataDialog from './components/InputDataDialog.js';
+import { initialInputData } from "./db";
+
+
+
 export default function App() {
-  return <div>App goes here</div>;
+  
+  const [inputs, setInputs] = useState(initialInputData);
+  
+  let actualDate = new Date();
+  let formattedActualDate = actualDate.toLocaleDateString()
+
+  return (
+    <>
+    <H1>Energy-Budget-App</H1>
+    <Main>
+      <section><p>{formattedActualDate}</p></section>
+      <ul>
+      {inputs.map(({ date, value, id}) => (
+          <li key={id}>
+            <span>{date}  {value} kw</span>
+          </li>
```

Comment:
I think that you do not need the `span` here. You can show the date directly in your `li` element.

```suggestion
          <li key={id}> {date}  {value} kw</li>
```